test/integration/features/basic.feature

```diff
@@ -35,8 +41,12 @@ Feature: Basic
       export "FOO=BAR"
       export "BAZ=BAT"
       """
-
+  Scenario: Stopping Minishift
+    Given Minishift should have state "Running"
```

Comment:
@hferentschik found out my reply disappeared somehow. I am wondering if we also want to remove should from for example this:

     Then stderr should be empty
      And exitcode should equal 0
      And stdout should contain